tests/appsec/test_traces.py

```diff
@@ -334,3 +334,53 @@ class Test_DistributedTraceInfo:
 
     def test_main(self):
         assert False, "Test not implemented"
+
+
+@rfc("https://docs.google.com/document/d/1xf-s6PtSr6heZxmO_QLUtcFzY_X_rT94lRXNq6-Ghws/edit?pli=1")
+class Test_ExternalWafRequestsIdentification:
```

Comment:
You misses the `@features` decorator. Ping me if need any help

Reply:
Done